src/test_l1_dsVideoDevice.c

```diff
@@ -405,11 +406,11 @@ void test_l1_dsVideoDevice_positive_dsSetDFC (void)
     UT_ASSERT_TRUE(handle >= 0);
 
     // 03-15: Set DFC mode with various zoom modes
-    for (int i = 0; i < dsVIDEO_ZOOM_MAX; ++i) 
-    {
-        dsVideoZoom_t zoomMode = i;
-        result = dsSetDFC(handle, zoomMode);
-        UT_ASSERT_EQUAL(result, dsERR_NONE);
+    for (int i = 0; i < gDSvideoDevice_NumVideoDevices; ++i) {
```

Comment:
this should be under loop i 
retStatus = dsGetVideoDevice(i, &handle);
Please take care of the rest of the functions also 
please check the l2 code

Reply:
done